src/pages/profile/basic/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { Badge, Card, Descriptions, Divider, Table } from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { connect } from 'dva';
+import { BasicProfileDataType } from './data.d';
+import styles from './style.less';
+
+const progressColumns = [
+  {
+    title: '时间',
+    dataIndex: 'time',
+    key: 'time',
+  },
+  {
+    title: '当前进度',
+    dataIndex: 'rate',
+    key: 'rate',
+  },
+  {
+    title: '状态',
+    dataIndex: 'status',
+    key: 'status',
+    render: (text: string) => {
+      if (text === 'success') {
+        return <Badge status="success" text="成功" />;
+      }
+      return <Badge status="processing" text="进行中" />;
+    },
+  },
+
+  {
+    title: '操作员ID',
+    dataIndex: 'operator',
+    key: 'operator',
+  },
+  {
+    title: '耗时',
+    dataIndex: 'cost',
+    key: 'cost',
+  },
+];
+
+interface BasicProps {
+  loading: boolean;
+  dispatch: Dispatch<any>;
+  profileAndbasic: BasicProfileDataType;
+}
+interface BasicState {
+  visible: boolean;
+}
+
+@connect(
+  ({
+    profileAndbasic,
+    loading,
+  }: {
+    profileAndbasic: BasicProfileDataType;
+    loading: {
+      effects: { [key: string]: boolean };
+    };
+  }) => ({
+    profileAndbasic,
+    loading: loading.effects['profileAndbasic/fetchBasic'],
+  }),
+)
+class Basic extends Component<BasicProps, BasicState> {
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'profileAndbasic/fetchBasic',
+    });
+  }
+
+  render() {
+    const { profileAndbasic, loading } = this.props;
+    const { basicGoods, basicProgress } = profileAndbasic;
+    let goodsData: typeof basicGoods = [];
+    if (basicGoods.length) {
+      let num = 0;
+      let amount = 0;
+      basicGoods.forEach(item => {
+        num += Number(item.num);
+        amount += Number(item.amount);
+      });
+      goodsData = basicGoods.concat({
+        id: '总计',
+        num,
+        amount,
+      });
+    }
+    const renderContent = (value: any, row: any, index: any) => {
+      const obj: {
+        children: any;
+        props: { colSpan?: number };
+      } = {
+        children: value,
+        props: {},
+      };
+      if (index === basicGoods.length) {
+        obj.props.colSpan = 0;
+      }
+      return obj;
+    };
+    const goodsColumns = [
+      {
+        title: '商品编号',
+        dataIndex: 'id',
+        key: 'id',
+        render: (text: React.ReactNode, row: any, index: number) => {
+          if (index < basicGoods.length) {
+            return <a href="">{text}</a>;
+          }
+          return {
+            children: <span style={{ fontWeight: 600 }}>总计</span>,
+            props: {
+              colSpan: 4,
+            },
+          };
+        },
+      },
+      {
+        title: '商品名称',
+        dataIndex: 'name',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'dataIndex' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)